### PR TITLE
p5-pdl: update to version 2.19.0

### PR DESCRIPTION
**Pull request by dbevans**
* add missing dependencies
* configure to build with non-standard proj4 install prefix

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by pmetzger** [comment]
As long as you're in there, can you add the maintainer's github handle?

File: perl/p5-pdl/Portfile [hunk 1]
```diff
@@ -4,9 +4,8 @@ PortSystem          1.0
 PortGroup           perl5 1.0
 
 perl5.branches      5.26
-perl5.setup         PDL 2.018
-revision            3
-maintainers         peculier.com:mgrimes openmaintainer
+perl5.setup         PDL 2.019
+maintainers         {peculier.com:mgrimes @mvgrimes}  openmaintainer
```
Comment (pmetzger) [inline perl/p5-pdl/Portfile]: Looks like there's an extra space before `openmaintainer` but maybe I'm being too picky. Overall, LGTM.

Comment (dbevans) [reply perl/p5-pdl/Portfile]: Even _port lint --nitpick_ missed that one.  Thanks.

**Comment by pmetzger** [comment]
Merged. Thanks, @dbevans!

**Pull request merged**